src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1138,6 +1147,37 @@ public function scopes($scopes)
         return $builder;
     }
 
+    /**
+     * Apply the default sorting to the Eloquent builder instance and return it.
+     *
+     * @return static
+     */
+    public function applyOrderBy()
+    {
+        if (! empty($this->orders)) {
+            return $this;
+        }
+
+        if (empty($this->model->orderBy)) {
```

Comment:
Shouldn't this line also be checking $this->defaultOrdering?

Reply:
No. It has been removed.

Reply:
Otherwise @jasonmccreary it looks like a great new approach.

Just a thought - are there any other places where applyOrderBy()->applyScopes() need chaining? e.g. `toBase()` ?

Reply:
> No. It has been removed.

So what will cover the situation where a query needs to be run on the model without any ordering (including the default ordering)?

Reply:
Potentially. I commented on this but there are probably more places this could be applied. I tried to draw the line between the Eloquent and Database query builders.

Since `toBase` returns a database query builder, I figured the user knows what they're doing and would not want this Eloquent sorting applied to their query. 🤷 

Pretty easy to add though, just makes change reach farther.